display_list/display_list_color_filter.h

```diff
@@ -57,6 +57,11 @@ class DlColorFilter
   // pixels non-transparent and therefore expand the bounds.
   virtual bool modifies_transparent_black() const = 0;
 
+  // Return a boolean indicating whether the color filtering operation can
+  // be applied either before or after modulating the pixels with an opacity
+  // value without changing the operation.
+  virtual bool can_commute_with_alpha() const { return false; }
```

Comment:
I think `kSrgbToLinearGamma` and `kLinearGammaToSrgb` also can commute with alpha, I verified it recently.

Reply:
I marked those as commutable when I added the test cases to verify commutability in the dl_canvas_unittests.cc